TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py

```diff
@@ -0,0 +1,543 @@
+"""
+    Wrapper functions for FuncIntent
+    This functions include Onosclidriver and Mininetclidriver driver functions
+"""
+def __init__( self ):
+    self.default = ''
+
+def hostIntent( main,
+                name="",
+                host1="",
+                host2="",
+                host1Id="",
+                host2Id="",
+                mac1="",
+                mac2="",
+                vlan1="-1",
+                vlan2="-1",
+                sw1="",
+                sw2="",
+                expectedLink=0 ):
+    """
+        Add host intents
+    """
+    import time
+
+    # Assert variables
+    assert main, "There is no main variable"
+    assert name, "variable name is empty"
+    assert host1 and host2, "You must specify hosts"
+
+    global itemName
```

Comment:
Yes, To all the main.log.info( itemName : ...) so I can easily see which
part of the intent test it is in the log.

On Tue, May 19, 2015 at 11:01 AM, Jon Hall <notifications@github.com> wrote:

> In TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/117#discussion_r30627603>
> :
>
> > +                vlan1="-1",
> > +                vlan2="-1",
> > +                sw1="",
> > +                sw2="",
> > +                expectedLink=0 ):
> > +    """
> > +        Add host intents
> > +    """
> > +    import time
> > +
> > +    # Assert variables
> > +    assert main, "There is no main variable"
> > +    assert name, "variable name is empty"
> > +    assert host1 and host2, "You must specify hosts"
> > +
> > +    global itemName
>
> Does this need to be global? where is this needed outside this function?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/117/files#r30627603>.
>